src/emailmessage.h

```diff
@@ -68,8 +72,20 @@ class Q_DECL_EXPORT EmailMessage : public QObject
         Redirect            = 5,
         UnspecifiedResponse = 6
     };
+
+    enum CalendarStatus {
```

Comment:
This should probably be named CalendarInvitationStatus, but actually it could be something more general like AttachedDataStatus.